File: BackEnd-Siukat/models/bobot.go
package models

type Bobot struct {
	ID      int     `gorm:"primaryKey;column:id" json:"id"`
	A       float64 `gorm:"column:a;type:decimal(10,2)" json:"a"`
	B       float64 `gorm:"column:b;type:decimal(10,2)" json:"b"`
	C       float64 `gorm:"column:c;type:decimal(10,2)" json:"c"`
	D       float64 `gorm:"column:d;type:decimal(10,2)" json:"d"`
	E       float64 `gorm:"column:e;type:decimal(10,2)" json:"e"`
	Pbb     float64 `gorm:"column:pbb;type:decimal(10,2)" json:"pbb"`
	Kontrak float64 `gorm:"column:kontrak;type:decimal(10,2)" json:"kontrak"`
}

func (Bobot) TableName() string {
	return "ref_bobot"
}

File: ZhammDataBeLama/siukat-service-dev_snmptn_2022/models/bobot.js
var Sequelize = require('sequelize')
var db = require('../config/database')

const Bobot = db.define('bobot', {
    id:{
        type: Sequelize.INTEGER,
        primaryKey: true
    },
    a:{
        type: Sequelize.DECIMAL
    },
    b:{
        type: Sequelize.DECIMAL
    },
    c:{
        type: Sequelize.DECIMAL
    },
    d:{
        type: Sequelize.DECIMAL
    },
    e:{
        type: Sequelize.DECIMAL
    },
    pbb:{
        type: Sequelize.DECIMAL
    },
    kontrak:{
        type: Sequelize.DECIMAL
    },
},
 {
  // don't add the timestamp attributes (updatedAt, createdAt)
  timestamps: false,

  // don't delete database entries but set the newly added attribute deletedAt
  // to the current date (when deletion was done). paranoid will only work if
  // timestamps are enabled
  paranoid: true,

  // don't use camelcase for automatically added attributes but underscore style
  // so updatedAt will be updated_at
  underscored: true,

  // disable the modification of tablenames; By default, sequelize will automatically
  // transform all passed model names (first parameter of define) into plural.
  // if you don't want that, set the following
  freezeTableName: true,

  // define the table's name
  tableName: 'ref_bobot'
}
);
module.exports = Bobot
